larpandora/LArPandoraInterface/LArPandoraGeometry.cxx

```diff
@@ -137,6 +185,32 @@ namespace lar_pandora {
 
   //------------------------------------------------------------------------------------------------------------------------------------------
 
+  unsigned int 
+  LArPandoraGeometry::GetDaughterVolumeID(const LArDriftVolumeMap &driftVolumeMap, 
+                                          const unsigned int cstat, 
+                                          const unsigned int tpc)
```

Comment:
Looks like this might fit within the standard Pandora character limit on one line? If not, combine any overflow on the following line and ensure indentation is only 1 "tab" wide